### PR TITLE
Update Release Schedule + Minor fixes

### PR DESCRIPTION
**Pull request by MariusBluem**
* Updated release schedule (EOL for `17` and `18` & release of `20`)
* Calendar-documentation is more about the `CalDAV`-backend - so lets clarify this in the title.
* Link to the correct desktop client documentation (`3.1` instead of `2.6`)

Signed-off-by: Marius Blüm <marius@lineone.eu>

### CODE REVIEW

**Comment by MariusBluem** [comment]
Commit https://github.com/nextcloud/documentation/pull/6095/commits/a156dced46a4199467f3ce365b9ad83faa3312d2 updates the `index.php`, especially:

* Footer was different to nextcloud.com-footer. Links are now updated.
* Copyright information was still 2019 ... updated it to 2021 - happy new year 🎉
* demo-link was updated to new subdomain try.nextcloud.com (see issue https://github.com/nextcloud/nextcloud.com/issues/1412)

**Comment by MariusBluem** [comment]
@nextcloud/documentation

File: admin_manual/groupware/calendar.rst [hunk 1]
```diff
@@ -31,8 +31,8 @@ Make sure the "Send notifications for events" and the "Enable notifications for
 Background jobs
 ~~~~~~~~~~~~~~~
 Running background jobs can be an expensive task when there are a large number of events, reminders, event sharees and attendees. However, this needs to happen
-often enough so that the notifications are sent on time. To accomplish this you should use a dedicated occ command that runs
-more often than the standard cron system::
+often enough so that the notifications are sent on time. To accomplish this you should use a dedicated ``occ`` command that runs
+more often than the standard `cron`` system::
```
Comment (kesselb) [inline admin_manual/groupware/calendar.rst:35]:
```suggestion
more often than the standard ``cron`` system::
```

or without

Comment (MariusBluem) [reply admin_manual/groupware/calendar.rst:35]: 👍

File: index.html [hunk 2]
```diff
@@ -439,12 +438,14 @@ <h1>About Nextcloud</h1>
 				<div class="footer-nav">
 					<h1>Resources</h1>
 					<ul>
+						<li><a class="reference external" href="https://nextcloud.com/search/">Search our website</a></li>
 						<li><a class="reference external" href="https://nextcloud.com/install">Download</a></li>
-						<li><a class="reference external" href="https://apps.nextcloud.com">App Store</a></li>
-						<li><a class="reference external" href="https://docs.nextcloud.com">Documentation</a></li>
-						<li><a class="reference external" href="https://nextcloud.com/enterprise">Enterprise Subscription</a></li>
+						<li><a class="reference external" href="https://docs.nextcloud.com/server/20/admin_manual/">Admin manual</a></li>
```
[the remark this answers is not in the document]
Comment (MariusBluem) [reply index.html:443]: See: https://github.com/nextcloud/nextcloud.com/blob/1d1f1fd5a09ac6828c5b706d23d673063cc1a7bd/strings.php#L15

Maybe we should also fix this over there but this does not seem to be there for no reason, since dev and user-manual is linking to `latest`.

[the remark this answers is not in the document]
Comment (MariusBluem) [reply index.html:443]: Maybe @jospoortvliet can tell us why this was done :)

**Review by LukasReschke — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -439,12 +438,14 @@ <h1>About Nextcloud</h1>
 				<div class="footer-nav">
 					<h1>Resources</h1>
 					<ul>
+						<li><a class="reference external" href="https://nextcloud.com/search/">Search our website</a></li>
 						<li><a class="reference external" href="https://nextcloud.com/install">Download</a></li>
-						<li><a class="reference external" href="https://apps.nextcloud.com">App Store</a></li>
```
Comment (rakekniven) [inline index.html:443]: Hello @MariusBluem ,

why did you remove the links here?

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (jospoortvliet) [reply index.html:443]: At some point, 'latest' was broken, but if it works, it's better I'd say ;-)

Sorry for the late reply.

[the remark this answers is not in the document]
Comment (jospoortvliet) [reply index.html:443]: oh and I should then update it on the website, too...

[the remark this answers is not in the document]
Comment (jospoortvliet) [reply index.html:443]: Done that.

File: index.html (hunk 3 above)
Comment (jospoortvliet) [reply index.html:443]: yeah, I wouldn't.

**Review by jospoortvliet — APPROVED** [review]
It'd be nice to bring back the app store link in the footer, but even without that - I'm good with merging this.

**Comment by rakekniven** [comment]
> It'd be nice to bring back the app store link in the footer, but even without that - I'm good with merging this.

Added it.
